Foundation/NSSwiftRuntime.swift

```diff
@@ -14,10 +14,16 @@ import CoreFoundation
 // This mimics the behavior of the swift sdk overlay on Darwin
 #if os(OSX) || os(iOS)
 @_exported import Darwin
-#elseif os(Linux)
+#elseif os(Linux) //// || os(Android) // Causes ambiguity in NSScanner.swift
```

Comment:
These constants are known in the standard library as greatestFiniteMagnitude and leastFiniteMagnitude. The spellings max and min were considered and explicitly rejected, and any such additions to the public API would have to go through the evolution process.